tests/v2/test_waku_rln_relay.nim

```diff
@@ -628,9 +634,84 @@ suite "Waku rln relay":
 
     # verify the proof (should not be verified)
     let verified = rln.proofVerify(data = messageBytes,
-                                 proof = proof)
+                                  proof = proof)
+    check:
+      verified.isOk()
+    check:
+      verified.value() == false
```

Comment:
Ditto here.

Reply:
Addressed in [fefb7e7](https://github.com/status-im/nwaku/pull/1158/commits/fefb7e706182ee964835d1393c5a9f7dc480ad9d)